ferveo-python/ferveo/__init__.pyi

```diff
@@ -101,34 +106,33 @@ class Dkg:
     def aggregate_transcripts(self, messages: Sequence[ValidatorMessage]) -> AggregatedTranscript:
         ...
 
-
+@final
 class Ciphertext:
     @staticmethod
-    def from_bytes(data: bytes) -> Ciphertext:
+    def from_bytes(bytes: bytes) -> Ciphertext:
         ...
 
     def __bytes__(self) -> bytes:
         ...
 
-
+@final
 class DecryptionShareSimple:
     @staticmethod
-    def from_bytes(data: bytes) -> DecryptionShareSimple:
+    def from_bytes(bytes: bytes) -> DecryptionShareSimple:
         ...
 
     def __bytes__(self) -> bytes:
         ...
-
-
+@final
 class DecryptionSharePrecomputed:
     @staticmethod
-    def from_bytes(data: bytes) -> DecryptionSharePrecomputed:
+    def from_bytes(bytes: bytes) -> DecryptionSharePrecomputed:
```

Comment:
I made an identical comment in https://github.com/nucypher/nucypher-core/pull/72, but I'd prefer to avoid name shadowing `bytes()` throughout this changeset.